src/AndorSIF.jl

```diff
@@ -184,20 +190,27 @@ function FileIO.load(fs::Stream{format"AndorSIF"})
     for i = 1:frames
         _ = readline(io)
     end
-    offset = position(io) # start of the actual pixel data, 32-bit float, little-endian
 
-    pixels = read(io, Float32, width, height, frames)
+    # two additional lines?
+    _ = readline(io)
+    _ = readline(io)
+
+    offset = position(io) # start of the actual pixel data, 32-bit float, little-endian
+    numbytes = 4 * frames * width * height # number of bytes to read
+    pixelbytes = read(io, numbytes)
+    pixelarray = reinterpret(Float32, pixelbytes)
+    pixelmatrix = reshape(pixelarray, (width, height))
     prop = Compat.@compat Dict(
         "colorspace" => "Gray",
-        "spatialorder" => ["y", "x"],
+        "spatialorder" => ["x", "y"],
```

Comment:
This is pretty outdated now. I'd use `permuteddimsview` from ImageCore.jl. Likewise, returning `Gray{Float32}` takes care of the colorspace field. Finally, `[1, 1]` is the default pixelspacing, better not to include it since folks may want to provide the correct answer (e.g., in microns) using Unitful.jl and AxisArrays.jl. So the only need for the `ImageMeta` is the `ixon` header.